packages/backend/src/graphql/scalars/BigInt.ts

```diff
@@ -23,10 +23,12 @@ function patchBigInt() {
   }
 }
 
-function coerceBigIntValue(value: bigint | number | string): bigint | number {
+function coerceBigIntValue(
+  value: bigint | number | string | unknown
+): bigint | number {
   if (isBigIntAvailable()) {
     patchBigInt()
-    const uInt64 = BigInt(value)
+    const uInt64 = BigInt(value as bigint | number | string)
```

Comment:
Should we be doing validation on `value`?
https://www.apollographql.com/docs/apollo-server/schema/custom-scalars/#example-restricting-integers-to-odd-values

Reply:
Probably